File: system-test-dotnet/SmokeTests/SmokeTest.cs
using Microsoft.Playwright;

namespace Optivem.AtddAccelerator.Template.SystemTest.SmokeTests
{
    public class SmokeTest
    {
        [Fact]
        public async Task ShouldOpenLocalhostAndCheckStatus200()
        {
            using var playwright = await Playwright.CreateAsync();
            await using var browser = await playwright.Chromium.LaunchAsync();
            var page = await browser.NewPageAsync();
            var response = await page.GotoAsync("http://localhost:8080");
            Assert.Equal(200, response.Status);
            await browser.CloseAsync();
        }
    }
}

File: system-test-java/src/test/java/com/optivem/atddaccelerator/template/systemtest/smoketest/SmokeTest.java
package com.optivem.atddaccelerator.template.systemtest.smoketest;

import com.microsoft.playwright.*;
import org.junit.jupiter.api.*;

import static org.junit.jupiter.api.Assertions.assertEquals;
import static org.junit.jupiter.api.Assertions.assertTrue;

public class SmokeTest {
    @Test
    void shouldOpenLocalhostAndCheckStatus200() {
        try (Playwright playwright = Playwright.create()) {
            Browser browser = playwright.chromium().launch();
            Page page = browser.newPage();
            Response response = page.navigate("http://localhost:8080");
            assertEquals(200, response.status());
            browser.close();
        }
    }
}
